experimental/MatroidRealizationSpaces/src/MatroidRealizationSpaces.jl

```diff
@@ -11,9 +14,45 @@ function underlying_scheme(RS::MatroidRealizationSpace{BRT, RT}) where {BRT<:Rin
   return RS.underlying_scheme::AffineScheme{BRT, RT}
 end
 
+function underlying_scheme(RS::MatroidRealizationSpace{BRT, RT}) where {BRT<:Ring, RT<:MPolyQuoRing}
+  isdefined(RS, :underlying_scheme) && return RS.underlying_scheme::AffineScheme{BRT, RT}
+
+  RS.underlying_scheme = spec(ambient_ring(RS)) # for some reason this is an MPolyQuoRing already
+  return RS.underlying_scheme::AffineScheme{BRT, RT}
+end
+
+
+
 # Exports
 export inequations
 export is_realizable
 export realization
 export realization_matrix
 export realization_space
+export defining_ideal
+
+#Exports from SelfProjectingMatroids.jl
+export is_selfprojecting
+export satisfies_disjointbasisproperty
+export selfprojecting_realization_space
+export selfproj_realization_ideal
+export selfproj_realization_matrix
+export inequations
+export defining_ideal
+export basisminors
+
+#Exports from for_database.jl
+export matroid 
```

Comment:
you shouldn't export functions that have already been exported.
sam for some functions above

Reply:
yes, how julia works is that a function has a list of methods (one for each type) and when you call the function it chooses the method. 
So since the function matroid is already exported, you don't need to export it again